src/zlist.c

```diff
@@ -394,14 +394,14 @@ zlist_sort (zlist_t *self, zlist_compare_fn *compare)
     bool swapped = false;
     while (gap > 1 || swapped) {
         if (gap > 1)
-          gap = (int) ((double) gap / 1.3);
+            gap = (int) ((double) gap / 1.3);
```

Comment:
Can you revert this part of the change?  @hintjens just sent in [a PR to normalize the whitespace](https://github.com/zeromq/czmq/pull/990), and he shouldn't have to do it again for the same line within a few hours :wink:.

See http://rfc.zeromq.org/spec:21 for applicable style guidelines.